test/acceptance/specs/ProblemReportUISpec.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.specs
+
+import acceptance.pages.{ProblemReportPage, ThankYouPage}
+import acceptance.specs.BaseSpec
+import acceptance.specs.tags.{UiTests}
+
+class ProblemReportUISpec extends BaseSpec {
+
+  info("Happy path for reporting a problem")
+
+  Feature("User reporting a problem") {
+
+    Scenario("A user reports a problem", UiTests) {
+
+      Given("A user is on the report a problem page")
+
+      go to ProblemReportPage
+      pageTitle shouldBe ProblemReportPage.title
+
+      ProblemReportPage.completeProblemReportForm
+
+      When("User submits the report")
+      ProblemReportPage.submitProblemReportForm
+
+      Then("The thank you page is displayed")
+      eventually {
+        pageTitle shouldBe ThankYouPage.title
+      }
+
+      ThankYouPage.heading shouldBe ThankYouPage.expectedHeading
+      ThankYouPage.subHeading shouldBe ThankYouPage.expectedSubHeading
```

Comment:
@natcarey this is probably tangential to the main focus of the review but after looking I feel I can't contribute much helpful feedback on some of the plumbing with sbt and structural conventions (other than that it makes sense to me what you've got!) so it's a really small test change but I thought I'd see what you thought about reducing some of these assertions into something like `on(ThankYouPage)` rather than matching page titles against members of the ThankYouPage - just ignore if these are copied from somewhere and being migrated in not written for the first time!

Reply:
I've been looking at this and I think it's a good suggestion but I think it's one that's best added on at a later time.